package.json

```diff
@@ -27,26 +27,26 @@
     "test:demo": "lerna exec -- yarn test:demo",
     "test:nonreg": "node screenshots.js",
     "test:extra": "yarn test:html-webpack-plugin && yarn test:logging",
-    "test:html-webpack-plugin": "cd packages/html-webpack-plugin && yarn && yarn test",
-    "test:logging": "cd packages/logging && yarn && yarn test",
+    "test:html-webpack-plugin": "cd packages/webpack-plugins/html-webpack-plugin && yarn && yarn test",
+    "test:logging": "cd packages/deprecated/logging && yarn && yarn test",
     "start": "echo Please use an other npm script like start-components",
     "start-components": "cd packages/components && yarn start",
     "start-containers": "cd packages/containers && yarn start",
     "start-datagrid": "cd packages/datagrid && yarn start",
-    "start-forms": "cd packages/forms && yarn start",
+    "start-forms": "cd packages/forms/react-forms && yarn start",
     "start-theme": "cd packages/theme && yarn start",
     "publish": "lerna publish --force-publish=*",
     "build-storybook": "npm run test:demo && export ACTION=\"test:demo\" && export TRAVIS_BRANCH=\"master\" && export TRAVIS_BUILD_DIR=`pwd` && .travis/after_success_static.sh && .travis/after_success_demo.sh && .travis/after_success_coverage.sh",
     "publish-storybook": "npm run build-storybook && cd .static && surge",
     "changelog": "git log --pretty=\"format:%C(bold green)%ad%C(reset) %<(100,trunc)%s\" --date=short --color",
     "extract-i18n": "npm run extract-i18n-components && npm run extract-i18n-forms && npm run extract-i18n-containers && npm run extract-i18n-datagrid",
     "extract-i18n-components": "i18next-scanner --config packages/components/i18next-scanner.config.js 'packages/components/src/**/*.js'",
-    "extract-i18n-forms": "i18next-scanner --config packages/forms/i18next-scanner.config.js 'packages/forms/src/**/*.js'",
+    "extract-i18n-forms": "i18next-scanner --config packages/forms/i18next-scanner.config.js 'packages/forms/react-forms/src/**/*.js'",
     "extract-i18n-containers": "i18next-scanner --config packages/containers/i18next-scanner.config.js 'packages/containers/src/**/*.js'",
     "extract-i18n-datagrid": "i18next-scanner --config packages/datagrid/i18next-scanner.config.js 'packages/datagrid/src/**/*.js'",
-    "audit-router": "cd packages/router && yarn audit --json && cd ../../",
-    "audit-scripts": "cd packages/scripts && yarn audit -s --json && cd ../../",
-    "audit-html": "cd packages/html-webpack-plugin && yarn audit --json && cd ../../"
+    "audit-router": "cd packages/cmf/router && yarn audit --json && cd ../../",
+    "audit-scripts": "cd packages/deprecated/scripts && yarn audit -s --json && cd ../../",
```

Comment:
please drop this one; it is depreacted now

Reply:
Done, thanks